### ISSUE TITLE
[Deque Analysis] Dark badges/buttons have insufficient color contrast

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1156]
## Deque Analysis Summary

In several locations, dark buttons (with white text) have insufficient color contrast.  Most of these (possibly all) may be using `btn-secondary` or `badge-secondary` which just needs to be made darker.

"Element has insufficient color contrast of 4.47 (foreground color: `#ffffff`, background color: `#777777`.  Contrast must be 4.5 or more."

1. Browse by Author (https://demo7.dspace.org/browse/author) - The author count badges next to each entry.  Also the "Settings" button (allowing you to change sort/results per page)
    * Deque issue ticket IDs: 469337
2. Submission (https://demo7.dspace.org/submit) - The "Save for Later" button (and "Save" when enabled)
    * Deque issue ticket IDs: 469531
3. Edit Item - Bitstreams Tab  - The "Back" button
    * Deque issue ticket IDs: 471057

## Recommended Fix

It looks like fixing this might be as simple as setting `btn-secondary` and `badge-secondary` to a slightly darker color like `#757575`

## More Information / Tools

* As necessary, see mentioned Issue IDs in our assessment results (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by tdonohue** [comment]
@artlowel : Assigning this to you for analysis.  Might be a quick fix (make the color darker)

**Comment by artlowel** [comment]
This will be fixed if we put the default bootstrap grays back, as the proposed solution for #1154

**Comment by tdonohue** [comment]
@artlowel : sounds good, I'll mark this as a zero hour estimate then as it'll be fixed by #1154
